src/Symfony/Component/Yaml/Inline.php

```diff
@@ -28,6 +28,7 @@ class Inline
     private static $exceptionOnInvalidType = false;
     private static $objectSupport = false;
     private static $objectForMap = false;
+    private static $constantSupport = false;
```

Comment:
Throwing the exception already depends on the ``exceptionOnInvalidType`` flag currently